src/search/components/shotNumber.component.tsx

```diff
@@ -92,6 +105,31 @@ const ShotNumber = (props: ShotNumberProps): React.ReactElement => {
   const invalidRange =
     min !== undefined && max !== undefined ? min > max : false;
 
+  const [flashAnimationPlaying, setFlashAnimationPlaying] =
+    React.useState<boolean>(false);
+
+  // Stop the flash animation from playing after 1500ms
+  React.useEffect(() => {
+    if (
+      (!props.searchParameterShotnumMax && !props.searchParameterShotnumMin) ||
```

Comment:
Need to use proper `typeof x !== undefined` checks here, as these falsy checks can trigger when min and max are 0 (which are valid values but causes the flash to happen)